percy/__init__.py

```diff
@@ -14,7 +14,7 @@ def percySnapshot(browser, *a, **kw):
 
 # import screenshot command
 try:
-    from percy.screenshot import percy_screenshot
+    from percy.snapshot import percy_screenshot
```

Comment:
This seems to be weird. Hope this is not breaking anything

Reply:
there is no module/file named screenshot so this already be giving error.

Reply:
@chinmay-browserstack  was this checked?

Reply:
yes renamed function

Reply:
FYI: currently not renaming percy_screenshot. will handle when making change in appium package as function name must be percy_screenshot

Reply:
We discussed this @itsjwala  it will need changes in both the packages. Idea is to have a `percy_screenshot` function and then call the required function called `app_percy_appium_screenshot`, `percy_selenium_screenshot`